plugins/hls-explicit-record-fields-plugin/src/Ide/Plugin/ExplicitFields.hs

```diff
@@ -154,7 +142,8 @@ collectRecordsRule recorder = define (cmapWithPrio LogShake recorder) $ \Collect
       recs = concat $ maybeToList (getRecords <$> tmr)
   logWith recorder Debug (LogCollectedRecords recs)
   let renderedRecs = traverse renderRecordInfo recs
-      recMap = buildIntervalMap <$> renderedRecs
+      -- All spans are supposed to be `RealSrcSpan`, hence the use of `fromJust`.
```

Comment:
there is no `fromJust`? Which we should never use anyway.

Reply:
Ah, right. There used to be a `fromJust` but I was dissatisfied with that so removed it along the way. Seems like I forgot to remove the comment...